src/auth/index.js

```diff
@@ -543,4 +543,39 @@ utils.wrapPropertyMethod(
  */
 utils.wrapPropertyMethod(AuthenticationClient, 'passwordGrant', 'oauth.passwordGrant');
 
+/**
+ * Sign in using a refresh token
+ *
+ * @method    refresh
+ * @memberOf  module:auth.AuthenticationClient.prototype
+ *
+ * @example <caption>
+ *   Given a refresh token from a previous authentication request
+ *   it will return a JSON with the access_token and id_token.
+ *   More information in the
+ *   <a href="https://auth0.com/docs/api/authentication#refresh-token">
+ *     API Docs
+ *   </a>.
+ * </caption>
+ *
+ * var data = {
+ *   client_id: '{CLIENT_ID}',  // Optional field.
+ *   refresh_token: '{REFRESH_TOKEN}',
+ * };
+ *
+ * auth0.refresh(data, function (err, userData) {
```

Comment:
I'm sorry, I should've catch this before, but let's change this to `refreshToken`. Thanks 🎉

Reply:
No worries. Updated now.

Reply:
Sorry again 😢 I was talking about the method name 😬  from `refresh` to `refreshToken`. The `refresh_token` parameter has to remain the same so the API understands it.